modules/storage_table_entity/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">=0.14.9"
+
+  required_providers {
+    azurerm = "3.6.0"
+  }
+
+  backend "azurerm" {}
+}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_storage_table_entity" "storage_table_entity" {
+  table_name           = var.storage_table_name
+  storage_account_name = var.storage_account_name
+
+  partition_key = var.partition_key
+  row_key       = var.row_key
+
+  entity = var.entity
+
+  lifecycle {
+    ignore_changes = [
+      entity
+    ]
```

Comment:
yes, We do not want to override this value if it was already created. 

For instance, 

We Created a entry it has time 2021-01-21, and this value is stored in state. Now the logic app changes the value to 2022-07-28. Since the new value does not update the state. Whenever we will want to re-apply terraform it will think that it has to set it back to 2021-01-21, which screws all of our previous runs